content/en/security/cloud_security_management/identity_risks/_index.md

```diff
@@ -43,12 +43,18 @@ Click **View Suggested Policy** to view a suggested downsized policy based on th
 
 {{< img src="security/identity_risks/downsized_policy.png" alt="Review suggestions for downsizing a policy on the Suggested downsized policy dialog" width="100%">}}
 
-To remediate the identity risk, click **Fix in AWS** to update the resource in AWS IAM console. To create a Jira issue and assign it to a team, click **Create Jira issue**. See [Create Jira Issues for Cloud Security Management Issues][2] for more information.
+To remediate the identity risk, click **Fix in AWS** to update the resource in AWS IAM console. To create a Jira issue and assign it to a team, click **Add Jira issue**. See [Create Jira Issues for Cloud Security Management Issues][2] for more information.
 
 {{< img src="security/identity_risks/side_panel_action_buttons.png" alt="Remediate identity risks using the action buttons on the side panel" width="100%">}}
 
 You can also leverage [Workflow Automation][3] to create automated workflows for identity risks (with or without human involvement). See [Automate Security Workflows with Workflow Automation][3] for more information.
 
+## AWS IAM Access Analyzer integration
+
+Datadog CIEM is integrated with [AWS IAM Access Analyzer][5] to further improve the permissions gap detections. If you are using AWS IAM Access Analyzer, Datadog CIEM automatically leverages its unused access findings to enrich permissions gap detections and downsized policy recommendations.
+
+Note: If you are enabling AWS IAM Access Analyzer for the first time, there is additional AWS cost associated with this enablement and it could take up to two hours before AWS IAM Access Analyzer's insights are made available.
```

Comment:
```suggestion
<div class="alert alert-info">If you are enabling AWS IAM Access Analyzer for the first time, there is an additional AWS cost associated with this enablement and it could take up to two hours before AWS IAM Access Analyzer's insights are made available.</div>
```